fhservices.cfg.j2

```diff
@@ -57,26 +57,26 @@ define host {
        host_name {{ rhmap_router_dns }}
 }
 
-{% for fhservice in fh_services_ping %}
+{% for fhservice in fh_services if 'ping' in fhservice['checks'] %}
 define service {
-       service_description {{ fhservice }}::Ping
+       service_description {{ fhservice['name'] }}::Ping
        hosts {{ rhmap_router_dns }}
-       check_command check_fh_component_http_ping!{{ fhservice }}
+       check_command check_fh_component_http_ping!{{ fhservice['name'] }}!{{ fhservice['port'] if 'port' in fhservice  else '8080' }}!{{ fhservice['ping_endpoint'] if 'ping_endpoint' in fhservice  else '/sys/info/ping' }}
        use generic-service
 
-       notes This server cannot access {{ fhservice }}, check it is running and that this server is able to talk to it on port 8080
+       notes This server cannot access {{ fhservice['name'] }}, check it is running and that this server is able to talk to it on port 8080
        contact_groups rhmapadmins
 }
 {% endfor %}
 
-{% for fhservice in fh_services_health %}
+{% for fhservice in fh_services if 'health' in fhservice['checks'] %}
 define service {
-       service_description {{ fhservice }}::Health
+       service_description {{ fhservice['name'] }}::Health
        hosts {{ rhmap_router_dns }}
-       check_command check_fh_component_http_health!{{ fhservice }}
+       check_command check_fh_component_health!{{ fhservice['name'] }}!{{ fhservice['port'] if 'port' in fhservice  else '8080' }}!{{ fhservice['health_endpoint'] if 'health_endpoint' in fhservice  else '/sys/info/health' }}
```

Comment:
I'm not sure what is valid in the jinja template, but perhaps if supports calling dict methods?

```python
fhservice.get("port", "8080")
fhservice.get("health_endpoint", "/sys/info/health")
```